dbms/src/Storages/DeltaMerge/Filter/PushDownExecutor.cpp

```diff
@@ -192,6 +195,8 @@ PushDownExecutorPtr PushDownExecutor::build(
         table_column_defines,
         context.getSettingsRef().dt_enable_rough_set_filter,
         tracing_logger);
+    // build column_range
+    const auto column_range = rs_operator && used_indexes.empty() ? rs_operator->buildSets(used_indexes) : nullptr;
```

Comment:
```suggestion
    const auto column_range = (rs_operator && !used_indexes.empty()) ? rs_operator->buildSets(used_indexes) : nullptr;
```
Should it be "not empty"?